### PR TITLE
Blog Post Build Modern Login/Signup Form using TailwindCSS and React

### PR DESCRIPTION
**Pull request by FuzzySid**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

### CODE REVIEW

File: content/guest-post/modern-login-signup-form-tailwindcss-react/index.md [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+You can use [LoginRadius](https://www.loginradius.com/) to quickly setup an authentication backend for your frontend application. First, you'll need to setup an account and get your API key and secret from your account Dashboard. Then, you can simply use their [login endpoint](https://api.loginradius.com/identity/v2/auth/login) to authenticate users. 
+
+The endpoint takes an email and password as parameters and validates them against them against the correct credentials. If we were to integrate this enpoint in our previously build `<Login/>` component (`/scr/components/Login.js`) here's, how the `authenticateUser` method would look like:
+
+```javascript
+//Handle Login API Integration here
+    const authenticateUser = () =>{
+        
+       
+        const endpoint=`https://api.loginradius.com/identity/v2/auth/login?apikey=${apiKey}&apisecret=${apiSecret}`;
```
Comment (VershaGupta) [inline content/guest-post/modern-login-signup-form-tailwindcss-react/index.md:754]: @FuzzySid  No need to pass apisecret for authentication apis. Refer Document : https://www.loginradius.com/docs/api/v2/customer-identity-api/authentication/auth-login-by-email/

Comment (VershaGupta) [inline content/guest-post/modern-login-signup-form-tailwindcss-react/index.md:754]: @FuzzySid  No need to pass apisecret for authentication apis. Refer Document : https://www.loginradius.com/docs/api/v2/customer-identity-api/authentication/auth-login-by-email/

Comment (VershaGupta) [inline content/guest-post/modern-login-signup-form-tailwindcss-react/index.md:754]: @FuzzySid  No need to pass apisecret for authentication apis. Refer Document : https://www.loginradius.com/docs/api/v2/customer-identity-api/authentication/auth-login-by-email/

**Review by VershaGupta — APPROVED** [review]
Approved

**Comment by raghunath-r-a** [comment]
This is duplicated/replaced by https://github.com/LoginRadius/engineering-portal/pull/1092

**Pull request closed without merging**